src/FufillerLogo.jsx

```diff
@@ -1,44 +1,79 @@
 import React, { Component } from 'react';
 import VisibilitySensor from 'react-visibility-sensor';
+import PropTypes from 'prop-types';
 
 export default class FulfillerLogo extends React.Component {
 
-  constructor(props) {
+  constructor (props) {
     super(props);
-    this.url = `https://fulfilleridentity.trdlnk.cimpress.io/v1/fulfillers/${props.fulfillerId}/logo`;
     this.state = {
       imageBlob: null,
+      url: `https://fulfilleridentity.trdlnk.cimpress.io/v1/fulfillers/${props.fulfillerId}/logo`,
+      visible: false,
+      imageIsLoading: false,
+      imageIsForbidden: false
+    };
+  }
+
+  componentWillReceiveProps (newProps) {
+    if (newProps.fulfillerId !== this.props.fulfillerId) {
+      this.setState({
+        imageBlob: null,
+        url: `https://fulfilleridentity.trdlnk.cimpress.io/v1/fulfillers/${newProps.fulfillerId}/logo`
```

Comment:
I lack some context here. Is the intent to update the component with the information for a new fulfiller and the component to start pointing to the new fulfiller's logo? We don't seem to store the fulfiller id anywhere except he in a roundabout way, correct?

It might be a good idea to store just the fulfiller id and construct the URL on demand when we need it, in case we need to use the fulfiller id for something else (COAM, etc.). Or not - it's just a thought.